models/article-models.js

```diff
@@ -1,12 +1,13 @@
 const db = require("../db/connection");
 
-exports.articleById = (article_id) => {
+exports.articleById = (article_id, comment_count) => {
   return db
     .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
     .then((article) => {
       if (article.rows.length === 0) {
         return Promise.reject({ status: 404, msg: "Article not found" });
```

Comment:
You also need to check the reason for the query returning empty rows, as this could be one of two reasons:
article exists but has no comments.
OR
article does not exist.
In order to determine the reason why, you will need to check whether the article exists first.